packages/web3-eth-abi/src/api/parameters_api.ts

```diff
@@ -147,6 +213,105 @@ export const decodeParametersWith = (
 /**
  * Should be used to decode list of params
  */
+/**
+ * Decodes ABI encoded parameters to its JavaScript types.
+ * @param abi -  An array of {@link AbiInput}. See the [Solidity documentation](https://docs.soliditylang.org/en/develop/types.html) for a list of types.
+ * @param bytes - The ABI byte code to decode
+ * @returns - The result object containing the decoded parameters.
+ * @example
+ * ```ts
+ * let res = web3.eth.abi.decodeParameters(
+ *    ["string", "uint256"],
+ *    "0x000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000ea000000000000000000000000000000000000000000000000000000000000000848656c6c6f212521000000000000000000000000000000000000000000000000"
+ *  );
+ *  > { '0': 'Hello!%!', '1': 234n, __length__: 2 }
```

Comment:
tiny nitpick, Instead of mix of `console logs` and few example without it, we should be consistent with one option?

Reply:
just missed the console in the copy paste, thanks you